### ISSUE TITLE
Setting for Tuner Go to sleep 

### ISSUE THREAD

**Issue by SaulGoodman1337** [Barracuda09/SATPI#172]
Can we have a setting where we can specify in seconds when satpi will put the tuners into sleep mode after no tuner is active ?
Currently it looks like the tuners do not go into sleep mode at all.
With minisatip it worked great though.

**Comment by Barracuda09** [comment]
Hi @SaulGoodman1337 

Thanks for your request!
Do you know how does that works? or how can I check this in Engima2 (I presume).

Regards,
Marc

**Comment by SaulGoodman1337** [comment]
I think you'll find what you need here, right? 
https://www.kernel.org/doc/html/v4.8/media/kapi/dtv-core.html

there is something that sounds like it

```
int (*)(struct dvb_frontend *fe) sleep
    callback function used to put the tuner to sleep.
```

does this help?

**Comment by SaulGoodman1337** [comment]
or this helps ? 
https://www.kernel.org/doc/html/v4.9/media/uapi/dvb/fe-set-voltage.html

**Comment by SaulGoodman1337** [comment]
And Here in enigma2 frontend.cpp
https://github.com/OpenPLi/enigma2/blob/04ae50fa033ffd689f7ec4227caa385bad2f7990/lib/dvb/frontend.cpp#L2725

**Comment by Barracuda09** [comment]
Hi @SaulGoodman1337 

Yes many thanks for your input, so basically it should turn the power off to the LNB after an adjustable time?

**Comment by SaulGoodman1337** [comment]
Yes, exactly. In my case, that's 6 watts that are uselessly consumed when no one is watching. 
Thank you for an effort!

**Comment by Barracuda09** [comment]
Yes , I understand the situation. I will try to look into this as soon as possible.

Thanks for you feedback!

**Comment by SaulGoodman1337** [comment]
Awesome thanks. Can you possibly provide me with a binary for a gigablue 4k?
